### DISCUSSION TITLE
How do I get a 12 word seed instead of 28 or 29?

### DISCUSSION THREAD

**Discussion by hackschnitzel09** [SiaFoundation/hostd#201] [Questions & Support] [answered]
hi, 
How do I get a 12 word seed instead of 28 or 29?

**Comment by n8mgr** [comment, marked as answer]
Follow one of the setup guides. They cover how to get a seed. https://docs.sia.tech/v/current/hosting/setup-guides

**Reply by hackschnitzel09** [reply]
Thank you :)
